a3p-integration/proposals/n:upgrade-next/agoric-tools.js

```diff
@@ -9,10 +9,14 @@ import { makeMarshal, Remotable } from '@endo/marshal';
 
 export const GOV4ADDR = 'agoric1c9gyu460lu70rtcdp95vummd6032psmpdx7wdy';
 
-const slotToRemotable = (_slotId, iface = 'Remotable') =>
-  Remotable(iface, undefined, {
-    getBoardId: () => _slotId,
-  });
+const slotToRemotable = (_slotId, iface = 'Remotable') => {
+  /** @type {BoardRemote} */
+  const methods = Object.defineProperties(
+    {},
+    { getBoardId: { value: () => _slotId } },
```

Comment:
Not a blocker but I've found the presence of `getBoardId` in the logs helpful to be clarify how it's different from regular cap data.

Reply:
I would hope that the debug tag suffices for that, although my big complaint about the noise here is mostly addressed by switching away from `@endo/init/legacy.js`:
```diff
-getBoardId: [Function: getBoardId] Function <Function <Complex prototype>>
+getBoardId: {}
```
I don't like that functions render as if they were plain objects, but that's a Node.js issue: https://github.com/nodejs/node/issues/55924

Still, I think I could now go either way on this.

Reply:
Agree completely to stop legacy.js. 

If you're okay with removing this getBoardId change then please do

Reply:
Done.